core/src/com/unciv/logic/automation/ChooseBeliefsAutomation.kt

```diff
@@ -155,31 +176,36 @@ object ChooseBeliefsAutomation {
                 UniqueType.BuyUnitsByProductionCost ->
                     15f * if (civInfo.wantsToFocusOn(Victory.Focus.Military)) 2f else 1f
                 UniqueType.StatsWhenSpreading ->
-                    unique.params[0].toInt() / 5f
+                    unique.params[0].toFloat() / 5f
                 UniqueType.StatsWhenAdoptingReligion, UniqueType.StatsWhenAdoptingReligionSpeed ->
                     unique.stats.values.sum() / 50f
                 UniqueType.RestingPointOfCityStatesFollowingReligionChange ->
-                    unique.params[0].toInt() / 7f
+                    if (civInfo.wantsToFocusOn(Victory.Focus.CityStates))
+                        unique.params[0].toFloat() / 3.5f
+                    else
+                        unique.params[0].toFloat() / 7f
                 UniqueType.StatsFromGlobalCitiesFollowingReligion ->
-                    50f / unique.stats.values.sum()
+                    unique.stats.values.sum()
```

Comment:
Dividing by the sum of stats causes the AI to evaluate beliefs that give more stats as worse so this has been changed to be just a simple sum.